test/api-type-document.test.js

```diff
@@ -229,6 +285,22 @@ describe('<api-type-document>', function() {
           assert.typeOf(element.andTypes, 'array');
           assert.lengthOf(element.andTypes, 3);
         });
+
+        it('Sets andTypes for ArrayShape with "and" property', async () => {
+          const data = await AmfLoader.load(item[1], 'APIC-429');
+          element.amf = data[0];
+          const payload = getResponseSchema(
+            element,
+            data[0],
+            '/pets',
+            'get',
+            '200'
+          );
+          element._typeChanged(element._resolve(payload[0]));
+          assert.isTrue(element.isArray);
```

Comment:
I believe this is incorrect. The `isArray` renders the template for the array view. It shows nothing now but the template is still in the DOM. Should be removed to reduce proc/mem usage. This should also test whether `isAnd` is set.